<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(77, 76, 125);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>-20</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Menu Ouvidoria ABC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 19, 68);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Descrição</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(35, 19, 68);</string>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Reclamação</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sugestão</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Elogio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 19, 68);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tipo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
border-radius: 10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
border-radius: 10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LISTAR TODAS AS MANIFESTAÇÕES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CRIAR MANIFESTAÇÃO</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>556</y>
      <width>2000</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(130, 115, 151);
color: none;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
border-radius: 10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LISTAR TODAS AS SUGESTÕES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
border-radius: 10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LISTAR TODAS AS RECLAMAÇÕES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
border-radius: 10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>LISTAR TODOS ELOGIOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(77, 76, 125);
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PROCURAR POR ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>660</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(255, 2, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>INSIRA UM ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>710</y>
      <width>86</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(54, 48, 98);
border-radius: 10px;
color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PROCURAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
